src/styles/app.scss

```diff
@@ -616,3 +616,13 @@ $border-colour: govuk-colour("grey-2");
   margin: 0; 
 }
 
+.govuk-fieldset{
+  &.horizontalChoices{
+    .govuk-radios{
+      display: inline-flex;
```

Comment:
Perhaps raise a story to review what other radio field sets should be changed to this new style.